.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+  "arrowParens": "avoid",
+  "bracketSpacing": true,
+  "bracketSameLine": true,
+  "printWidth": 160,
+  "singleQuote": true,
+  "trailingComma": "none"
```

Comment:
Controversial opinion, but I like having trailing commas. If nothing else it's good for keeping diffs concise.

Reply:
I like them as well. We can be both cause controversy together!

Reply:
I wouldn't be opposed.  I instinctively hate it, but the advantage on diffs is probably enough to allow my brain to eventually rewire itself and get used to it.  Would definitely result in a lot more file changes in this repo, though.

Reply:
Yeah I'm fine with enabling this if everyone wants to. I agree with @MikeMatusz, I'm not a fan because the noise it adds to the files and it makes things feel incomplete to me but the diff reason is a good point. I think we should try it without them for now, and if we feel the need to enable it later we can do so.